src/components/SVGViewer/__tests__/SVGViewer.test.tsx

```diff
@@ -53,4 +53,31 @@ describe('SVGViewer', () => {
     expect(wrapper.find(CANCEL_BUTTON_ATTR).length).toBe(0);
     expect(wrapper).toHaveLength(1);
   });
+
+  it('updates maxZoom and minZoom props after creation', done => {
+    const defaultProps = {
+      documentId: 0,
+      ...SVGViewer.defaultProps,
+    };
+    const wrapper = mount(
+      <ClientSDKProvider client={sdk}>
+        <SVGViewer documentId={0} />
+      </ClientSDKProvider>
+    );
+    expect(wrapper.props().children.props).toEqual(defaultProps);
+    wrapper.setProps(
+      { children: <SVGViewer documentId={0} maxZoom={50} minZoom={10} /> },
+      () => {
+        setImmediate(() => {
+          wrapper.update();
+          expect(wrapper.props().children.props).toEqual({
+            documentId: 0,
+            maxZoom: 50,
+            minZoom: 10,
```

Comment:
but this doesn't test anything?

you still don't know if `pinchZoomInstance` gets them?

Reply:
I agree, how about this approach:

```javascript
    const SVGViewerComponent = wrapper.find(SVGViewer).instance() as SVGViewer;
    sinon.stub(SVGViewerComponent.pinchZoom.current as any, 'appendChild');
    SVGViewerComponent.initPinchToZoom();
    const { options } = SVGViewerComponent.pinchZoomInstance;
    expect(options.maxZoom).toEqual(50);
    expect(options.minZoom).toEqual(10);
```

Reply:
I've cleaned up test a bit so we see the main point

```
t('updates maxZoom and minZoom props after creation', () => {
    const wrapper = mount(
      <ClientSDKProvider client={sdk}>
        <SVGViewer file="<svg/>" />
      </ClientSDKProvider>
    );
    const SVGViewerComponent = wrapper.find(SVGViewer).instance() as SVGViewer;
    sinon.stub(SVGViewerComponent.pinchZoom.current as any, 'appendChild');
    wrapper.setProps(
      { children: <SVGViewer documentId={0} maxZoom={50} minZoom={10} /> },
      () => {
        SVGViewerComponent.initPinchToZoom(); // is this a bug? we shouldn't be calling it explicitly
        const { options } = SVGViewerComponent.pinchZoomInstance;
        expect(options.maxZoom).toEqual(50); // no, it's still  30
        expect(options.minZoom).toEqual(10); // still 1
      }
    );
  });
```

and happens because we don't call `this.presetSVG()` in `componentDidUpdate`.
Or am I missing something?
You guys probably know more about this component, so I might be mistaking

@valgussev it might be okay if viewer doesn't react on maxZoom/minZoom idk

Reply:
It is weird to call the method explicitly, there was a problem with `componentDidUpdate` that didn't update the `pinchZoomInstance`. It should be fixed now.